runtime/standalone/src/lib.rs

```diff
@@ -89,6 +117,8 @@ construct_runtime!(
         Contracts: pallet_3vm_contracts = 120,
         Evm: pallet_3vm_evm = 121,
         AccountManager: pallet_account_manager = 125,
+
+        Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 122,
```

Comment:
what if we add more vms? Ideally this should be near the system pallets at the top

Reply:
hahaha, that make sense!